lib/config/yml_generator_config.dart

```diff
@@ -342,4 +362,32 @@ class YmlGeneratorConfig {
           'Could not generate all models. `$type` is not added to the config file, but is extended. These types are known: ${names.join(',')}');
     }
   }
+
+  ItemType _parseSimpleType(String type) {
+    final listRegex = RegExp(r'List<\s*([^<>]*)\s*>');
+    final mapRegex = RegExp(r'Map<([^<>,]*)\s*,\s*([^<>,]*)\s*>');
+
+    if (type == 'object' || type == 'dynamic' || type == 'any') {
```

Comment:
Why not lowercase `type` so `Object` is valid, you don't need a separate check for `String`, `DateTime` is valid, etc 
(we should also do that in `_makeGenericName`)

Reply:
We can do this indeed, but need to keep in mind that List<Bla> is not the same as list<bla>

Reply:
Can also be a ticket which we can pickup later ofc